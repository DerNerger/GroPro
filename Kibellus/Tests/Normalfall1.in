;**************************
;     Normalfall 1         
;**************************
     blank    
baer   
    rad