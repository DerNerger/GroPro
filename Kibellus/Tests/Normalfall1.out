;**************************
;     Normalfall 1         
;**************************
Eingelesene Woerter:
BLANK, BAER, RAD

** Raetsel nicht versteckt

BAER  
 BLANK
   D  

** Raetsel versteckt

BAERXT
IBLANK
UHCDWF

Kompaktheitsmass: 18